src/test/java/io/confluent/examples/streams/microservices/util/MicroserviceTestUtils.java

```diff
@@ -107,12 +108,12 @@ protected static Properties producerConfig(final EmbeddedSingleNodeKafkaCluster
 
     final List<KeyValue<K, V>> actualValues = new ArrayList<>();
     TestUtils.waitForCondition(() -> {
-      final ConsumerRecords<K, V> records = consumer.poll(100);
+      final ConsumerRecords<K, V> records = consumer.poll(Duration.ofMillis(100));
       for (final ConsumerRecord<K, V> record : records) {
         actualValues.add(KeyValue.pair(record.key(), record.value()));
       }
       return actualValues.size() == numberToRead;
-    }, 20000, "Timed out reading orders.");
+    }, 30000, "Timed out reading orders.");
```

Comment:
Generally I'm not a fan of increasing timeouts, but in this case, `20 seconds` seemed a little low as most of our timeouts are set to `30 seconds`